packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -529,8 +527,6 @@ const SavedChartsHeader: FC = () => {
                     opened={isMovingChart}
                     onClose={() => setIsMovingChart(false)}
                     onConfirm={() => {
-                        sessionStorage.removeItem('fromDashboard');
-                        sessionStorage.removeItem('dashboardUuid');
```

Comment:
I haven't tested this change ( waiting for PR env to be ready ) but I suspect this is broken now. Where is the storage being cleared for this scenario?

Reply:
Thats the one that gets cleared when arriving on the dashboard. 

One thing this change does is makes it so the dashboard clears the editing indicators when it's loaded. Before this change, every place that redirected back to the dashboard was clearing that indicator before the redirect (like here). But there are a lot of places you can go back to the dashboard: the save button, this button, a similar button in the toast, the browser back button. And some of them weren't doing it right so the header was broken. 

Instead of trying to keep all of those in sync, the dashboard will now clear that indicator on mount. That way it gets done correctly and you don't have to duplicate that operation everywhere. AND you never want those indicators there on the dashboard anyway.

Reply:
But this action doesn't redirect to the dashboard. This modal/action is used when the user wants to move the chart, that belongs to a dashboard, to a space. So the header should disappear. 

Another problem comes to mind: What happens if I'm editing a dashboard -> edit chart -> move chart to space -> ?? ( I lose the dashboard unsaved changes )

Reply:
Ah, I see, I thought this was the other one in this file. I will check on this one and make sure this PR doesn't break it. 

To your second point, there are a lot of these kinds of issues with unsaved changes. I can show you other ones like that 😄 . I'm trying to fix one issue at a time. This PR was for the header issue specifically, but losing unsaved changes is the next thing to tackle.

Reply:
Updated to fix this.